config/config.exs

```diff
@@ -0,0 +1,6 @@
+use Mix.Config
+
+if Mix.env() == :dev do
+  config :mix_test_watch,
```

Comment:
what does this do?

Reply:
It configures the mix_test_watch package for the dev environment, i.e. you can run `mix test.watch` to watch file changes and rerun tests automatically.

Reply:
(Or more precisely: the config is only needed because clearing the screen before re-running the tests is not a default configuration for the package. Else it would work out of the box, even without the config)